expr/builtins/hash_and_encode.go

```diff
@@ -186,3 +186,22 @@ func encodeB64DecodeEval(ctx expr.EvalContext, args []value.Value) (value.Value,
 	}
 	return value.NewStringValue(string(by)), true
 }
+
```

Comment:
Im going to talk this one over with the team tomorrow.  If the goal here is to have this generate a random salt, that is only set once and never changed.  We may need to write this as one of our internal built-ins, so it has context from the entire entity.  Otherwise each time the profile is evaluated it's salt will change, which will change it's hashed email address.